virt-v2v/cold/entrypoint.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+const (
+	OVA     = "ova"
+	vSphere = "vSphere"
+	DIR     = "/var/tmp/v2v"
+	FS      = "/mnt/disks/disk[0-9]*"
+	Block   = "/dev/block[0-9]*"
+	VDDK    = "/opt/vmware-vix-disklib-distrib"
+)
+
+func main() {
+	fmt.Println("Bella")
+	virtV2vArgs := []string{"virt-v2v", "-v", "-x"}
+	source := os.Getenv("V2V_source")
+
+	requiredEnvVars := map[string][]string{
+		vSphere: {"V2V_libvirtURL", "V2V_secretKey", "V2V_vmName"},
+		OVA:     {"V2V_diskPath", "V2V_vmName"},
+	}
+
+	if envVars, ok := requiredEnvVars[source]; ok {
+		if !checkEnvVariablesSet(envVars...) {
+			fmt.Printf("Following environment variables need to be defined: %v\n", envVars)
+			//klog.Errorf("Following environment variables need to be defined: %v\n", envVars)
```

Comment:
klog.errorf? fmt.Printf? in the whole file. i think logger with error is a better fit

Reply:
I want originally with klog but currently this dependency not working since virt-v2v set in a different workspace then the rest off the project, once i"ll be able to solve these issues i"ll update the logger as well.